qcore/geo.py

```diff
@@ -2,24 +2,27 @@
 Various tools which may be needed in various processes.
 """
 
-from math import sin, asin, cos, atan, atan2, degrees, radians, sqrt, pi
 import os
+import numpy as np
 from subprocess import Popen, PIPE
 
-import numpy as np
+from math import sin, asin, cos, atan, atan2, degrees, radians, sqrt, pi
 
 from qcore.config import qconfig
+
 ll2xy_bin = os.path.join(qconfig['tools_dir'], 'll2xy')
 xy2ll_bin = os.path.join(qconfig['tools_dir'], 'xy2ll')
 
 R_EARTH = 6378.139
 
+
 class InputError(Exception):
     pass
 
-def ll2gp_multi(coords, mlon, mlat, rot, nx, ny, hh, \
-        dx = 1, dy = 1, decimated = False, verbose = False, \
-        keep_outside = False):
+
+def ll2gp_multi(coords, mlon, mlat, rot, nx, ny, hh,
+                dx=1, dy=1, decimated=False, verbose=False,
+                keep_outside=False):
```

Comment:
just run "black code.py" (https://github.com/ambv/black) and it will format without bothering about it